src/compiler/diagnosticMessages.json

```diff
@@ -6354,18 +6354,14 @@
         "category": "Message",
         "code": 90012
     },
-    "Import '{0}' from module \"{1}\"": {
+    "Import '{0}' from \"{1}\"": {

```

Comment:
This is used for both add and update?  Also, nitpicky, but why one set of single quotes and one set of double quotes?

Reply:
I don't think I even see this message used the diff. Am I missing it?

Reply:
I missed it too - it's referred to by its property name in the tests.

Reply:
@amcasey yes, both add and update. We always use double quotes for module specifiers for some reason, and single quotes for other code quoting 🤷‍♂️